src/tests/nativeaot/nativeaot.csproj

```diff
@@ -1,8 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
-    <ItemGroup>
-      <MergedWrapperProjectReference Include="*/**/*.??proj" Exclude="*/**/Library.csproj" />
-    </ItemGroup>
-  
-    <Import Project="$(TestSourceDir)MergedTestRunner.targets" />
-  </Project>
+  <PropertyGroup>
+    <CLRTestExitCode Condition="'$(CLRTestExitCode)' == ''">100</CLRTestExitCode>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <MergedWrapperProjectReference Include="*/**/*.??proj" Exclude="*/**/Library.csproj;*/**/HelloWasm/*.??proj" Condition="'$(TargetsWasm)' != 'true'" />
+    <MergedWrapperProjectReference Include="*/**/*.??proj" Exclude="*/**/Library.csproj"  Condition="'$(TargetsWasm)' == 'true'" />
```

Comment:
IMO it would be better to "include everything" (basically upstream include), then exclude the WASM projects we care about, basically replicating the list we had in `src/tests/Common/dirs.proj`.

Also, `dirs.proj` itself needs to be updated to be synced to upstream version now that no changes are needed for it.

Reply:
We can't use this for Wasm as we are not using the merged wrapper, but have changed for non -wasm